lib/elixir_jobs_web/templates/offer/_offer.html.eex

```diff
@@ -33,7 +33,15 @@
       </span>
     </div>
   </div>
-  <div class="description">
-    <%= sanitized_markdown(@offer.description) %>
-  </div>
+  <%= if @details do %>
+    <div class="description">
+      <p> <%= @offer.summary %> </p>
```

Comment:
I would use the summary only in listings. i think it could be confusing for the people that send offers for the first time.

It looks (at least for me) more intuitive to have a summary and a full description, but not concatenating them. What do you think?

Reply:
I think with the suggested `placeholder` message for both `summary` and `description` it makes sense to not concatenate.